### DISCUSSION TITLE
How to simulate impedance transformer with openEMS?

### DISCUSSION THREAD

**Discussion by Sarajevo67** [thliebig/openEMS-Project#103] [Q&A Usage]
I couldn't find anything in Tutorials and Examples, that address simulation of known frequency/impedance vector through TL?

**Comment by JTSvejda** [comment]
Hello,

I‘m not sure what exactly you like to simulate. Do you like to simulate a impedance matching circuit like a quarter wavelength transformer?

Best,
Taro

**Reply by Sarajevo67** [reply]
Hello Taro,

It is described and answered in: #100 

BR,

DD
